src/components/sections/employeeHighlight/employeeHighlight.module.css

```diff
@@ -18,7 +18,7 @@
 .image {
   min-width: 20rem;
   width: 100%;
-  height: auto;
+  height: 24rem;
```

Comment:
Is this intended? Should height ALWAYS be set to 24rem? Why shouldn't it scale?

Reply:
In other words: What is this intended to solve by not having it dynaic

Reply:
Heheh fixed it now